testbook/client.py

```diff
@@ -128,10 +128,12 @@ def inject(self, code, args=None, prerun=None):
 
     def value(self, name):
         """Extract a JSON-able variable value from notebook kernel"""
-
-        result = self.inject(name)
-        if not self._execute_result(result.outputs):
-            raise ValueError('code provided does not produce execute_result')
+        try:
+            result = self.inject(name)
+            if not self._execute_result(result.outputs):
+                raise Exception('code provided does not produce execute_result')
+        except Exception as e:
```

Comment:
I'd put a comment that you also want to catch inject CellExecutionErrors this way

Reply:
I'm working on a commit that moves this try/except to `execute_cell` instead, which will take care of inject CellExecutionErrors as well.